src/components/AlertsList.vue

```diff
@@ -0,0 +1,86 @@
+<script setup lang="ts">
+import { ref, toRef, type Ref } from 'vue';
+import { type ZodFlawType } from '@/types/zodFlaw';
+import { useAlertsModel } from '@/composables/useAlertsModel';
+import AlertsSection from '@/components/AlertsSection.vue';
+import LabelCollapsible from './widgets/LabelCollapsible.vue';
+
+const props = defineProps<{
+  flaw: any;
+}>();
+
+const emit = defineEmits<{
+  'expandFocusedComponent': [value: string];
+}>();
+
+const emitExpandFocusedComponent = (parent_uuid: string) => {
+  emit('expandFocusedComponent', parent_uuid);
+};
+
+const flaw: Ref<ZodFlawType> = toRef(props, 'flaw');
```

Comment:
Is the reactivity working? So after saving the `refresh:flaw` also updates the alerts
I couldn't check it.